src/main/webapp/css/main.css

```diff
@@ -161,6 +161,83 @@ body {
   margin-top: 10px;
 }
 
+.messageContainer{
+  width:100%;
+  margin-bottom:10px;
+  overflow:hidden;
+  display:flex;
+}
+
+.right{
+  justify-content: flex-end;
+}
+
+.box {
+  background: #ffffff;
+  color: #007bff;
+  max-width: 30%;
+  margin-left:25px;
+  margin-top:10px;
+  margin-right:25px;
+  padding: 20px;
+  text-align: center;
+  font-weight: 900;
+  font-family: arial;
+  position:relative;
+  border-radius:10px;
+}
+
+.box.blue{
+  background: #007bff;
+  color: #ffffff;
+}
+
+.leftBubble:before {
```

Comment:
You can try to share the common rules with the next class, there's quite a few common lines.